src/html.ts

```diff
@@ -141,6 +145,14 @@ class HTMLRenderer {
   }
 
   renderAstNode(node: AstNode): void {
+    const override = this.options.overrides?.[node.tag];
```

Comment:
Amusingly, this even type-checks that I didn't miss any cases in the visitor. This also unearthed a bunch of inconsistentcies: blockqote, hardbreak, softbreak, symbol